sds_data_manager/constructs/api_gateway_construct.py

```diff
@@ -61,26 +66,26 @@ def __init__(
         if domain_construct is not None:
             custom_domain = apigw.DomainName(
                 self,
-                "RestAPI-DomainName",
-                domain_name=f"api.{domain_construct.domain_name}",
+                f"{self.prefix}RestAPI-DomainName",
+                domain_name=f"{self.prefix.lower()}api.{domain_construct.domain_name}",
```

Comment:
I prefer `ialirt.prod.imap-mission.com` to `ialirtapi.prod.imap-mission.com`. It may also make sense to use `ialirt.imap-mission.com` and `dev.ialirt.imap-mission.com`. `alirtapi` is not easy to type or read. 

Could also do something like `api.ialirt.imap-mission.com`...?